### ISSUE TITLE
Suggestion: automatic backup to the cloud or get data from the cloud

### ISSUE THREAD

**Issue by mariowitdoek** [mRemoteNG/mRemoteNG#1732]
It would be interesting if there was a backup possibility to backup the conf files to Google Drive, Onedrive, Nextcloud, ... or maybe a simple FTP server (like a NAS)

## Expected Behavior

I just reinstalled my computer ... without a backup :( ... 
Nor further comments please and yes it is a life lesson ...

## Possible Solution
via a kind of addin system with builtin scheduler?
or ... getting the data directly for off device ... so a reinstall or a multi location solution can be implemented

## Your Environment
Windows 10

**Comment by loftwah** [comment]
I agree with the idea but it adds complexity to the application. Why not just put the configuration file in a directory that you have backing up to the cloud already?

**Comment by cpshoemake** [comment]
I run the whole application from a Nextcloud synced directory. Just download the .zip file, extract it into whatever cloud service you like, and run it from there.

**Comment by farosch** [comment]
You should also be able to link to the config file with a [hardlink](https://docs.microsoft.com/de-de/windows-server/administration/windows-commands/mklink).

**Comment by robertpopa22** [comment]
Triage refresh (P4 version-label debt): this issue is tagged with legacy version labels. Please retest on the latest build (fork v1.79.x or upstream dev), then share reproducible steps and expected vs actual behavior. If still valid, keep open and relabel to current triage scope (for example Need 2 check). If fixed, close with resolution.

Current legacy version labels: 1.77.2

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
